internal/cli/alerts/describe.go

```diff
@@ -37,14 +37,18 @@ func (opts *DescribeOpts) initStore() error {
 	return err
 }
 
+var describeTemplate = `ID	TYPE_NAME	METRIC_NAME	STATUS
```

Comment:
[nit] let's keep this human friendly, ie, no underscores and better naming
```suggestion
var describeTemplate = `ID	TYPE	METRIC	STATUS
```